chcemvediet/apps/obligees/tests/test_views.py

```diff
@@ -143,18 +217,17 @@ def test_autocomplete_with_multiple_words(self):
         found = [d[u'obligee'][u'name'] for d in data]
         self.assertItemsEqual(found, [u'aaa bbb ccc'])
 
-    @unittest.skip(u'FIXME')
     def test_autocomplete_matches_obligee_name_prefixes(self):
         names = [u'aa', u'aaa', u'aaaaaaa', u'aaaxxxx', u'xxxxaaa', u'xxxxaaaxxxx', u'xxx aaa', u'xxx aaax xxx']
         oblgs = [self._create_obligee(name=n) for n in names]
         response = self.client.get(reverse(u'obligees:autocomplete') + u'?term=aaa')
         self.assertEqual(response.status_code, 200)
         data = json.loads(response.content)
         found = [d[u'obligee'][u'name'] for d in data]
-        self.assertItemsEqual(found, [u'aaa', u'aaaaaaa', u'aaaxxxx', u'xxx aaa', u'xxx aaax xxx'])
+        self.assertItemsEqual(found, [u'aaa', u'aaaaaaa', u'aaaxxxx', u'xxxxaaa', u'xxxxaaaxxxx', u'xxx aaa', u'xxx aaax xxx'])
```

Comment:
Tu treba premenovat test `test_autocomplete_matches_obligee_name_prefixes` na `test_autocomplete_matches_obligee_name_substrings`, kedze zjavne to nemachuje iba prefix, ale aj v strede slova.

Plus by sa medze vzorove `oblgs` zislo pridat viac takych, ktore nemachnut. Minimalne nieco ako `xxx`, kde nie je ziadne "a".

Reply:
:ok: